syllabus/7-Polkadot/13-polkadot-ecosystem-economy-slides.md

```diff
@@ -0,0 +1,652 @@
+---
+title: Polkadot Ecosystem and Economy
+description: A review of the parts of Polkadot which develop its ecosystem and economy.
+duration: 1 hour
+---
+
+# Ecosystem and Economy
+
+---
+
+## Ecosystem and Economy
+
+This presentation will give you a high level overview of the ecosystem and economy of the Polkadot Network.
+
+Unfortunately this presentation could never be fully exhaustive, but perhaps it will shed light on areas previously unknown.
+
+---
+
+# Economy
+
+---
+
+## The DOT Token
+
+<div class="grid grid-cols-2">
+
+<div>
+
+<image src="../../assets/img/7-Polkadot/eco/token.avif" style="width: 400px;">
+
+</div>
+
+<div class="text-left">
+
+The DOT token can be in one of the following states:
+
+1. Transferable
+2. Locked (Frozen)
+3. Reserved (Held)
+
+</div>
+
+</div>
+
+---
+
+## Reserved vs Locked Balance
+
+- New terms "Frozen" and "Held" are not quite used in Polkadot yet...
+- DOT in both states belong to the user... but cannot be spent / transferred.
+- Reserved balances stack on top of one another.
+  - Useful for user deposits, or other use cases where there is sybil concerns.
+  - Ex: Deposit for storing data on-chain,
+- Locked balances can overlap each other.
+  - Useful when you want to use the same tokens for multiple use cases.
+  - Ex: Using the same tokens for both staking and voting in governance.
+
+---
+
+## Storage Bloat
+
+One blockchain scaling problem is storage bloat over time.
+
+<br />
+
+Consider the "cost" of storing data on Ethereum:
+
+- A one time gas fee based on the amount of data stored.
+- Once is it placed on the network, it lives there forever, with no additional costs.
+- Over a long enough period of time, the cost of storage per time will reduce to zero.
+
+---
+
+## Storage Deposits
+
+To solve this problem, Polkadot additionally takes a storage deposit (in the form of Reserved Balance) for any data stored in the blockchain.
+
+- This deposit is returned to the user when the user removes the data from the chain.
+
+- This deposit can be quite extreme, since it is returned to the user, and can represent the impermanence or lack of "importance" of the data.
```

Comment:
A bit confused about this one. Where is that storage? It does not live on the relay chain, right? What are you referring to here? (maybe I'm missing something)
What data are we talking about here?